cmd/fleetctl/package_test.go

```diff
@@ -43,10 +45,74 @@ func TestPackage(t *testing.T) {
 	}
 
 	t.Run("deb", func(t *testing.T) {
-		runAppForTest(t, []string{"package", "--type=deb", "--insecure", "--disable-open-folder"})
+		shorterEnrollSecret := "aa"
+		longerEnrollSecret := "aaaaaa"
+		runAppForTest(t, []string{"package", "--type=deb", "--insecure", "--disable-open-folder", "--enroll-secret=" + longerEnrollSecret, "--fleet-url=https://localhost:8080"})
 		info, err := os.Stat(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
 		require.NoError(t, err)
-		require.Greater(t, info.Size(), int64(0)) // TODO verify contents
+		require.Greater(t, info.Size(), int64(0))
+
+		fd, err := os.Open(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
+		if err != nil {
+			panic(err)
+		}
+
+		debFile, err := deb.Load(fd, fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion))
+		if err != nil {
+			panic(err)
+		}
+		for {
+			hdr, err := debFile.Data.Next()
+			if err == io.EOF {
+				break
+			}
+			require.NoError(t, err)
+
+			if hdr.Name != "./etc/default/orbit" {
+				continue
+			}
+
+			data, err := io.ReadAll(debFile.Data)
+			require.NoError(t, err)
+
+			s := string(data)
+			require.Contains(t, s, fmt.Sprintf("ORBIT_ENROLL_SECRET=%s\n", longerEnrollSecret))
+		}
```

Comment:
To avoid duplication you could use a local function:
```go
checkEnrollSecret := func(filepath string, expectedEnrollSecret string) {
  // [...]
}

checkEnrollSecret(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion), longerEnrollSecret)
// [...]
checkEnrollSecret(fmt.Sprintf("fleet-osquery_%s_amd64.deb", updatesData.OrbitVersion), shorterEnrollSecret)
```